Zend/tests/cwd_is_slash_at.phpt

```diff
@@ -3,7 +3,7 @@ Test IS_SLASH_AT for multibyte path
 --SKIPIF--
 <?php 
 include "skipif.inc"; 
-if (substr(PHP_OS, 0, 3) != 'WIN' || extension_loaded('mbstring')) {
+if (substr(PHP_OS, 0, 3) != 'WIN' || !extension_loaded('mbstring')) {
 	die ("skip only for Windows");
```

Comment:
It should also skip if `chcp 932` fails, because it'll fail on incompatible systems. Say on usual US codepage it fails with the message "invalid code page".